addons/point_of_sale/tools/posbox/posbox_create_image.sh

```diff
@@ -24,6 +24,8 @@ require_command zerofree
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 __file="${__dir}/$(basename "${BASH_SOURCE[0]}")"
 __base="$(basename ${__file} .sh)"
+__version=11.0-posboxupgrade-jco
```

Comment:
set to 11

Reply:
Indeed, it is need to be changed before merging.